config/config.yml

```diff
@@ -8,5 +8,5 @@ service:
 
 dependencies:
     rm-comms-template-db:
-        schema: comms-template-service
+        schema: templatesvc
```

Comment:
I wonder if we should be moving away from using yml files and using .env @davidmort ?

Reply:
I would go with the .env approach if we are going to continue with pipenv - it's supported out of the box https://github.com/kennethreitz/pipenv#-features